safe/utilities/keyword_io.py

```diff
@@ -108,18 +101,7 @@ def __init__(self, layer=None):
         self.connection = None
         self.layer = layer
 
-    def set_keyword_db_path(self, path):
-        """Set the path for the keyword database (sqlite).
-
-        The file will be used to search for keywords for non local datasets.
-
-        :param path: A valid path to a sqlite database. The database does
-            not need to exist already, but the user should be able to write
-            to the path provided.
-        :type path: str
-        """
-        self.keyword_db_path = str(path)
-
+    # TODO(IS) can be removed
```

Comment:
That TODO is for `read_keywords_file` method. I want to delete it, but there is InaSAFE wrapper that use it.
Perhaps we can address it on the other PR.